src/cljs/org/broadinstitute/firecloud_ui/page/method_repo/methods_configs_acl.cljs

```diff
@@ -30,8 +30,7 @@
   {:render
    (fn [{:keys [props refs state this]}]
      [modal/OKCancelForm
-      {:header (let [sel-ent (:selected-entity props)]
-                 (str "Permissions for " (sel-ent "entityType") " " (get-ordered-name sel-ent)))
+      {:header (str "Permissions for " (:entityType props) " " (:entityName props))
```

Comment:
I suggest making a single prop describing what this is an ACL editor of, maybe called `:title`.